src/test/groovy/graphql/parser/ParserTest.groovy

```diff
@@ -516,17 +516,21 @@ class ParserTest extends Specification {
     }
 
 
-    def "parses null value"() {
+    def "parses null values"() {
         given:
-        def input = "{ foo(bar: null) }"
+        def input = "{ foo(bar: null, bell : null) }"
 
         when:
         def document = new Parser().parseDocument(input)
         def operation = document.definitions[0] as OperationDefinition
         def selection = operation.selectionSet.selections[0] as Field
 
         then:
-        selection.arguments[0].value == NullValue.Null
+        assert selection.arguments[0].value instanceof NullValue
+        assert selection.arguments[1].value instanceof NullValue
+
+        assert selection.arguments[0].sourceLocation.toString() == "SourceLocation{line=1, column=7}"
```

Comment:
don't use assert: spock just takes a == b
also: this is the right test, yes, but not sure how this works, see the other comment above

Reply:
Spock will fail a test when any statement in the `then:` block is non true

So `selection.arguments[0].value instanceof NullValue` is a valid statement - no need for assert

Assert is needed if you create a common method that you call in a test eg

```

when:
     someCommonMethos(value)

def someCommonMethod(value) {
    assert value != null
    assert value instanceof Foo
    return true
}
```